### PR TITLE
fixes issue 12: EPSG codes incorrect for southern hemisphere

### PR DESCRIPTION
**Pull request by pjhartzell**
- Added a test and test data. As with existing test data, the new test data is a full scene, which is heavy (~10MB).
- Formatters added new lines in several files.

### CODE REVIEW

File: src/stactools/landsat/mtl_metadata.py [hunk 1]
```diff
@@ -68,10 +68,8 @@ def processing_level(self) -> str:
     def epsg(self) -> int:
         utm_zone = self._root.find_text('PROJECTION_ATTRIBUTES/UTM_ZONE')
         if utm_zone:
-            bbox = self.bbox
             utm_zone = self._get_text('PROJECTION_ATTRIBUTES/UTM_ZONE')
-            center_lat = (bbox[1] + bbox[3]) / 2.0
-            return int(f"{326 if center_lat > 0 else 327}{utm_zone}")
+            return int(f"326{utm_zone}")
```
Comment (gadomski) [inline src/stactools/landsat/mtl_metadata.py:79]: Can you throw a comment in here about how the transform, as provided by USGS, requires the UTM zone to always be north, even if the data are in the southern hemisphere?

**Review by gadomski — CHANGES_REQUESTED** [review]
Instead of committing these ~100MB files, use the [get external data](https://github.com/stac-utils/stactools/blob/main/src/stactools/testing/test_data.py#L42) functionality from `stactools.testing` to fetch the files for testing.

File: src/stactools/landsat/mtl_metadata.py (hunk 1 above)
Comment (pjhartzell) [reply src/stactools/landsat/mtl_metadata.py:79]: Yep. Done.

**Comment by pjhartzell** [comment]
> Instead of committing these ~100MB files, use the [get external data](https://github.com/stac-utils/stactools/blob/main/src/stactools/testing/test_data.py#L42) functionality from `stactools.testing` to fetch the files for testing.

The original data is about 1 GB, which was a mistake on my part. I replaced those files with resampled data to match the existing test data, which is also resampled. This cuts it down to 6.5 MB, which is in line with the current test data (`assets` subdirectories in `tests/data-files`).

**Review by gadomski — APPROVED** [review]
How does PySTAC do with the v0.7.0 STAC?

**Comment by pjhartzell** [comment]
> How does PySTAC do with the v0.7.0 STAC?

OK? There are tests that successfully manipulate Items generated with `Item.from_file` from v0.7.0 STAC json files. The output looks reasonable as well.

**Pull request closed without merging**